.env.docker.example

```
@@ -41,3 +41,15 @@ M365_REDIRECT_URI=http://localhost:8080/api/auth/m365/callback
 SMB_USER=mirai\\administrator
 SMB_PASS=FILL_FROM_cert_GMSV0002_txt
+
+# ── CMDB PC端末同期 (Project-CMDB連携) ──────────────────────
+# PC端末情報を5分間隔でWDMS DBに取り込む
+# CMDB_SYNC_ENABLED=true にすると同期が開始されます
+CMDB_SYNC_ENABLED=false
+# D:\Project-CMDB\CMDB ディレクトリのパス (Dockerバインドマウント用)
+# デフォルト: ./cmdb-data (プロジェクト内プレースホルダー — 同期は機能しない)
+# 本番設定: CMDB_SYNC_ENABLED=true と合わせてProject-CMDBのパスを指定
+# ※ パス区切りはスラッシュ(/) を使用
+CMDB_DATA_PATH=D:/Project-CMDB/CMDB
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CMDB_DATA_PATH key should go before the CMDB_SYNC_ENABLED key

(UnorderedKey)